### PR TITLE
okd scos periodic missing upi-installer

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

**Comment by jcpowermac** [comment]
@vrutkovs @LorbusChris I keep seeing errors on the new vsphere build cluster about missing this image. I _think_ this fixes it.

/pj-rehearse periodic-ci-openshift-release-master-okd-scos-4.14-e2e-vsphere-ovn

File: ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+base_images:
+  upi-installer:
+    name: "4.12"
```
Comment (LorbusChris) [inline ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml:3]:
```suggestion
    name: "scos-4.12"
```
same change is needed on all of the others too

**Review by LorbusChris — CHANGES_REQUESTED** [review]
@jcpowermac thank you for taking a look at this.
I had noticed that too, and discovered we weren't building the upi-installer image for SCOS.

Builds for 4.14 and 4.15 were added in https://github.com/openshift/release/pull/43624, so it'll only work with these versions.

You'll have to reference the `scos-4.1x` ImageStream though for it to point to the correct image.

I'd propose to drop the change for 4.13 and below. I'll open a separate PR to drop any vsphere tests on these OKD/SCOS versions.

**Comment by LorbusChris** [comment]
@jcpowermac I hope you don't mind that I've folded this into https://github.com/openshift/release/pull/44273

/close

**Pull request closed without merging**